COMMON_ISSUES.md

```diff
@@ -179,6 +179,37 @@ before opening a new issue. We recommend regularly upgrading to the latest versi
 <p>
 
    Safari on iOS version 15, sometimes routes audio to the earpiece and not the speakers by default. Which customers some time perceive as low audio volume. Find more details [here](https://github.com/twilio/twilio-video.js/issues/1586) and in this [WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=230902).
+
+   As a workaround, you can pipe all remote audio tracks into a single audio context for iOS 15. Using a gain node, you can increase the gain value to increase the audio volume levels. See example below.
+
+   ```js
+   // Make sure to reuse the audioContext object as browsers
+   // have limits to the number of AudioContext instances you can create.
+   const audioContext = new (window.AudioContext || window.webkitAudioContext)();
+
+   function attachAudioTrack(track) {
```

Comment:
Is this a Twilio AudioTrack? It's not clear from the example.

Reply:
Yes it is. I'll rename the variable.